### ISSUE TITLE
Large output leads to semaphore exceeding limit crash.

### ISSUE THREAD

**Issue by fsword7** [lenticularis39/axpbox#38]
I tested some more.  I tried execute 'dir/full sys$system'.  It crashed in middle of large output results.  It scrolls so fast like in SIMH VAX emulator (almost same speed match).  Take a look on crash information at end.

Tim

------------------------------------

$ dir/full sys$system

:
:

OPENVMS$FTP.EXE;1             File ID:  (3763,1,0)            
Size:            6/16         Owner:    [SYSTEM]
Created:    21-DEC-2016 13:39:23.01
Modified:   21-DEC-2016 13:39:23.01 (1)
Expires:    <None specified>
Backup:     <No backup recorded>
Effective:  <None specified>
Recording:  <None specified>
Accessed:   17-NOV-2020 21:13:22.46
Attr Mod:   21-DEC-2016 13:39:23.01
Data Mod:   21-DEC-2016 13:39:23.01
Linkcount:  1
File organization:  Sequential
Shelved state:      Online 
Caching attribute:  Writethrough
File attributes:    Allocation: 16, Extend: 0, Global buffer count: 0
                    No version limit, Contiguous best try
Record format:      Fixed length 512 byte records
Record attributes:  None
RMS attributes:     None
Journaling enabled: None
File protection:    System:RWED, Owner:RWED, Group:RE, World:RE
Access Cntrl List:  None
Client attributes:  None

OPENVMS$FTPDIR.EXE;1                      File ID:  (3764,1,0)            
Size:            6/16         Owner:    [SYSTEM]
Created:    21-DEC-2016 13:39:23.06
Modified:   21-DEC-2016 13:39:23.07 (1)
Expires:    <None specified>
Backup:     <No backup recorded>
Effective:  <None specified>
Recording:  <None specified>
Accessed:   17-NOV-2020 21:13:22.49
Attr Mod:   21-DEC-2016 13:39:23.07
Data Mod:   21-DEC-2016 13:39:23.07
Linkcount:  1
File organization:  Sequential
Shelved state:      Online 
Caching attribute:  Writethrough
File attributes:    Allocation: 16, Extend: 0, Global buffer count: 0
                    No version limit, Contiguous best try
Record format:      Fixed length 512 byte records
Record attributes:  None
RMS attributes:     None
Journaling enabled: None
File protection:    System:RWED, Owner:RWED, Group:RE, World:RE
Access Cntrl List:  None
Connection closed by foreign host.
sword7@tsux1vm:~/axpbox/data$ 

------------------------------------------------------------------------------------

Exception in CPU thread: System exception: cannot signal semaphore: count would exceed maximum.
Emulator Failure: Threading error: CPU thread has died: /home/sword7/axpbox/src/src/AlphaCPU.cpp, line 517
Stop threads: cpu0 srl0 symException in SYM thread: Threading error: Locking error (Timeout: /home/sword7/axpbox/src/src/base/Mutex.cpp, line 124) trying to lock mutex sym-reg from thread <thread>.
: /home/sword7/axpbox/src/src/base/Mutex.cpp, line 128.
 ali kbd
Freeing memory in use by system...
pci0.1(sym53c810).disk0.0(file): Closing file.

**Comment by lenticularis39** [comment]
Thank you for this interesting observation - this is the first CPU-related bug I know of (besides the memtest skip hack, but that one I introduced myself).

I tried to reproduce this with the (slower) IDE controller with no success - trying again with the significantly faster SCSI controller.

**Comment by fsword7** [comment]
Yes, AXP emulator is enough fast.  I ran VUPS and got 63.0 as result on my PC with i7-8800K (z370 chipset).
